### PR TITLE
Add fancier QRs (and QR decoding) to MobFriend

### PR DESCRIPTION
**Pull request by itdaniher**
* Enables downloads (to /tmp) in Auxin.
* Adds zbar-py and amzqr as MobFriend deps. 
* Adds "_actually_build_wait_and_send_qr" to MobFriend.

Long term we'll want to replace amzqr with "something else" but this is not urgent. 

Takes slightly more time to build QRs for payment requests etc, but they're all fancy now!

┆Issue is synchronized with this [Asana task](https://app.asana.com/0/1201632126650504/1201632375538613) by [Unito](https://www.unito.io)

### CODE REVIEW

File: forest/core.py [hunk 1]
```diff
@@ -114,7 +114,7 @@ async def start_process(self) -> None:
             )
             if not utils.AUXIN:
                 path += " --trust-new-identities always"
-            command = f"{path} --config {utils.ROOT_DIR} --user {self.bot_number} jsonRpc".split()
+            command = f"{path} --config {utils.ROOT_DIR} --download-path /tmp --user {self.bot_number} jsonRpc".split()
```
Comment (technillogue) [inline forest/core.py:117]: needs to be only if utils.AUXIN

File: mobfriend/mobfriend.py [hunk 2]
```diff
@@ -23,10 +28,73 @@
...
+                    brightness=1.0,
+                    picture=image_path,
+                )
+            )
+        await self.send_message(user_id, "Building your QR code! Please be patient!")
+        p = aioprocessing.AioProcess(
+            target=amzqr.run, args=(text,), kwargs=default_params
+        )
+        p.start()
+        await p.coro_join()
```
Comment (technillogue) [inline mobfriend/mobfriend.py:80]: this should prevent deadlocks, right? have you noticed anything fishy on shutdown?

Comment (itdaniher) [reply mobfriend/mobfriend.py:80]: no issues with deadlocks, and no issues with shutdown / restart

File: forest/core.py (hunk 1 above)
Comment (itdaniher) [reply forest/core.py:117]: working on it

File: mobfriend/Pipfile [hunk 3]
```diff
@@ -11,6 +11,7 @@ phonenumbers = "*"
 termcolor = "*"
 prometheus_async = "*"
 prometheus_client = "*"
+mobilecoin = "*"
```
Comment (technillogue) [inline mobfriend/Pipfile:14]: this is the old package with mobilecoind bindings - we only kept it around for the protobufs, but that's covered under protobufs

Comment (itdaniher) [reply mobfriend/Pipfile:14]: needed to provide the right version of "google"

Comment (technillogue) [reply mobfriend/Pipfile:14]: i think you still want "protobuf"?

**Pull request merged**